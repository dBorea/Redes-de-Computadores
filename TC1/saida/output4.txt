sala instanciada com sucesso
sala instanciada com sucesso
sala instanciada com sucesso
sensores inicializados com sucesso
sensores inicializados com sucesso
salas: 0 (27 80 10 20 31 40) 4 (40 35 12 22 32 42) 7 (-1 -1 -1 -1 -1 -1)
