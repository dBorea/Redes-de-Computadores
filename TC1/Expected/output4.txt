sala instanciada com sucesso
sala instanciada com sucesso
sala instanciada com sucesso
sensores inicializados com sucesso
sensores inicializados com sucesso
salas: 4 (40 35 12 22 32 42) 0 (27 80 10 20 31 40) 7 (-1 -1 -1 -1 -1 -1)
